### DISCUSSION TITLE
RE: questions

### DISCUSSION THREAD

**Discussion by Joe23232** [osmandapp/OsmAnd#11491] [Q&A]
Hi, I have some questions regarding this project:
1. I understand that it uses OpenStreetMaps but is this particular app free and open source?
2. Does this app collect telemetry data? if so can I disable any form of tracking that the app may do?
3. Does it show alternative routes if determined it is faster while I am driving, kinda like what Google maps does?
4. Can I prevent toll routes? If a toll road has just been added in my area, how long would it take for OsmAnd to register the new toll road(s)?
5. What is the difference between the free and the paid version? What am I missing out if I go for the free version?
6. Does it alert me if there are speed cameras near me?
7. Can I install third party extensions?
8. Is there a dark mode that enables automatically when it gets dark?
9. Is this well supported in Australia?
10. If I am NOT going to a destination, can I still activate the maps and show me where I am?
11. If I change settings etc, would it be stored locally and not be synced to the cloud? If it is synced to the cloud would it use end to end encryption?
12. Is it lighter weight compared to Google Maps? Does it eat up less battery compared to Google Maps?
13. Do I get traffic alerts like with Google Maps if there is like 4 minutes congestion ahead of me?

**Comment by vshcherb** [comment]
Wow quite a lot of questions, probably some of them you can get answer by googling them and trying app how it looks. I would ike to start answering with first 4.
1. Open Source but not Free 
2. Doesn't collect anything at all, privacy focused app
3. Not alternative routes yet
4. Yes, you can add / edit toll roads on OpenStreetMap

**Reply by Joe23232** [reply]
I see thanks and one last thing would it warn me in regards to speed cameras?

**Reply by lefuturiste** [reply]
1. According to the GPL v3 license this *is* free software!!

**Reply by tommycrock** [reply]
> According to the GPL v3 license this _is_ free software!!

The English language specialises in ambiguity. It's libre but not gratis.

**Comment by sonora** [comment]
Some more answers:

ad 3) OsmAnd does not propose a choice of routes, but will always recalculate the route if you leave it, meaning you will always see the best route from your current locarion.

ad 5) OsmAnd+ offers e.g. unlimited map download from within the app, and can indicate places on the map which are described in wikipedia.org.

ad 6) If legal in your country, you can activate this. Of course the cams in question need to be contained in the openstreetmap.org map data.

ad 7) Which one?

ad 8) Yes. Why don't you just try the apo instead of asking stuff like this?

ad 9) Supported globally.

ad 10) Sure. Again, please try.

ad 11) Settings are stored locally.

ad 12) Google maps works mostly online, thus requiring a mobile Internet connection, which OsmAnd does not, hence OsmAnd does not spend battery power on cell connectovity. But OsmAnd does also not have online informationlike e.g. about ttaffic jans.

**Reply by Joe23232** [reply]
I see thanks
